src/main/java/httpserver/App.java

```diff
@@ -1,20 +1,23 @@
 package httpserver;
 
+import httpserver.route.RouteHandler;
 import httpserver.server.ConsoleWriter;
 import httpserver.server.Server;
 
 public class App {
     private static final int defaultPort = 5000;
 
     public static void main(String[] args) {
+        var routes = RouteConfig.create();
+        var routeHandler = new RouteHandler(routes);
         var port = checkPort(args);
-        var server = new Server(port);
+        var server = new Server(port, routeHandler);
 
         server.start();
     }
 
     private static int checkPort(String[] args) {
```

Comment:
This method is getting the port or selecting the port. Check doesn't suggest it's actually returning a value.

Reply:
Thanks, Matt. I'll rename this